requirements-torch-cuda.txt

```diff
@@ -4,8 +4,8 @@ tensorflow-text~=2.18
 
 # Torch with cuda support.
 --extra-index-url https://download.pytorch.org/whl/cu121
```

Comment:
you will likely need to mess with this line as well. you can play around with torch install lines on the torch home page. Looks like we need to choose from cuda 11.8, 12.4 and 12.6